devlib/target.py

```diff
@@ -494,7 +494,7 @@ def do_pull(sources, dest):
             for source in sources:
                 with self._xfer_cache_path(source) as device_tempfile:
                     self.execute("cp -r -- {} {}".format(quote(source), quote(device_tempfile)), as_root=True)
-                    self.execute("chmod 0644 -- {}".format(quote(device_tempfile)), as_root=True)
+                    self.execute("chmod 0644 {}".format(quote(device_tempfile)), as_root=True)
```

Comment:
Does the `busybox` version?

Reply:
Yes it does. Updated to use that instead.